vertx-web-client/src/main/asciidoc/index.adoc

```diff
@@ -372,6 +372,21 @@ that decode the response to a specific type
 
 WARNING: this is only valid for the response decoded as a buffer.
 
+=== Server-Sent Events
+
+Using a specific body codec: {@link io.vertx.ext.web.codec.sse.SseBodyCodec}, you can decode a Server-Sent Events stream into a list of events:
+
+[source,$lang]
+----
+client.get("/eventsource").as(SseBodyCodec.sseStream(stream -> {
+      stream.handler(event -> //process the event);
+      stream.endHandler(v -> {
+        async.complete();
+      });
+    })).send();
```

Comment:
Could you please move this to `WebClientExamples`? (see other examples in this docs)

Among other things, having examples in source files allow to make sure they still compile with future modifications